.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -1,6 +1,9 @@
 ---
 name: Feature request
 about: Suggest an idea for this project
+title: ''
+labels: ''
+assignees: ''
```

Comment:
See my comments above, I don't think we should include these if they are just the defaults.

Reply:
Same as above